DocFX unable to "compile" Oqtane client

Trying to compile the Octane 5.0 documents. Octane 5.0 compiles as a project just fine as well as the Oqtane Docs Builder solution once everything is updated to use .Net 8, but running the post build event **dotnet docfx** fails with the following error:

...\oqtane.framework\Oqtane.Client\Modules\ModuleBase.cs(40,16): error CS0246: The type or namespace name 'ModuleInstance' could not be found (are you missing a using directive or an assembly reference?)

Get same error running **dotnet docfx** in a command prompt window.

--- a/.config/dotnet-tools.json
+++ b/.config/dotnet-tools.json
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "docfx": {
-      "version": "2.67.3",
+      "version": "2.74.1",
       "commands": [
         "docfx"
       ]

--- a/.vscode/launch.json
+++ b/.vscode/launch.json
@@ -10,7 +10,7 @@
       "request": "launch",
       "preLaunchTask": "build",
       // If you have changed target frameworks, make sure to update the program path.
-      "program": "${workspaceFolder}/Oqtane.Docs/bin/Debug/net5.0/Oqtane.Docs.dll",
+      "program": "${workspaceFolder}/Oqtane.Docs/bin/Debug/net8.0/Oqtane.Docs.dll",
       "args": [],
       "cwd": "${workspaceFolder}/Oqtane.Docs",
       // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console

--- a/Oqtane.Docs/Oqtane.Docs.csproj
+++ b/Oqtane.Docs/Oqtane.Docs.csproj
@@ -1,10 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Library</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
-    <RestoreOnBuild>true</RestoreOnBuild>
-  </PropertyGroup>
-  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="dotnet docfx" />
-  </Target>
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>Library</OutputType>
+		<TargetFramework>net8.0</TargetFramework>
+		<RestoreOnBuild>true</RestoreOnBuild>
+	</PropertyGroup>
+	<Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+		<Exec Command="dotnet tool list -g | findstr docfx || dotnet tool install --global docfx" />
+	</Target>
+	<Target Name="PostBuild" AfterTargets="PostBuildEvent">
+		<Exec Command="dotnet docfx" />
+	</Target>
 </Project>

--- a/Oqtane.Docs/docfx.json
+++ b/Oqtane.Docs/docfx.json
@@ -71,7 +71,7 @@
     "overwrite": [
       {
         "files": [
-          "apidoc/**.md",
+          "apidoc/**.md"
         ],
         "exclude": [
           "obj/**",

--- a/Oqtane.Docs/docfx.json
+++ b/Oqtane.Docs/docfx.json
@@ -22,7 +22,7 @@
       "dest": "obj/api",
       "filter": "filterConfig.yml",
       "properties": {
-        "TargetFramework": "net7.0"
+        "TargetFramework": "net8.0"
       }
     }
   ],

--- a/Oqtane.Docs/index.md
+++ b/Oqtane.Docs/index.md
@@ -1,8 +1,8 @@
-# Oqtane Documentation 4.0.0
+# Oqtane Documentation 5.0.1
 
 > Oqtane is _the_ **Modular Application Framework for Blazor**
 >  
-> Modern, Flexible, and Open Source - running on .net 7.0!
+> Modern, Flexible, and Open Source - running on .net 8.0!
 
 ## Documentation Contents
 

